sleepypuppy/templates/admin/file/list.html

```diff
@@ -1,9 +1,9 @@
 {% extends 'admin/master.html' %}
 {% import 'admin/lib.html' as lib with context %}
-{% import 'admin/actions.html' as actionslib with context %}
+{% import 'admin/actions.html' as actionlib with context %}
 
 {% block body %}
-    {% block breadcrums %}
+    {% block breadcrumbs %}
```

Comment:
This doesn't appear referenced, but I have no way of knowing if it's a breaking change.